### ISSUE TITLE
How to continue the train of the existing model and How to interpret accuracy of model on test dataset?

### ISSUE THREAD

**Issue by talatccan** [qubvel/segmentation_models.pytorch#210]
Hi,

I've trained model with custom dataset following example tutorial. Results are very good.
I have two question in this stage. I get following result when evaluate on test set.  So should I interpret the success of the model as 98%? If not is there any way to get calculate accuracy of it?

![image](https://user-images.githubusercontent.com/10474206/82250349-f467cc80-9953-11ea-9d1c-03c313fbac78.png)

And if i want to continue training from last saved best model what should i do?

**Comment by JulienMaille** [comment]
You should load your checkpoint after creating your model, something like
```python
state_dict = torch.load('./best_model.pth')
model.load_state_dict(state_dict.state_dict())
```

**Comment by talatccan** [comment]
> You should load your checkpoint after creating your model, something like
> 
> ```python
> state_dict = torch.load('./best_model.pth')
> model.load_state_dict(state_dict.state_dict())
> ```

Thank you so much for reply. I will modify the code like this.
How can i interpret model accuracy? Is it correct to say model accuracy is 98% because IOU score is 0.98 on test set?
Is there any different or better metric to determine the model accuracy score? Or is there a way to create confusion matrix for binary segmentation?

**Comment by JulienMaille** [comment]
Intersection over union and accuracy are two different metrics provided by this library, you should Google for it, there are many good tutorials on this. If you need the confusion matrix you will have to implement it as it is not in the library

**Comment by talatccan** [comment]
Thank you so much.
